jsk_fetch_robot/jsk_fetch_startup/config/config.bash

```diff
@@ -0,0 +1,7 @@
+# This file is bash configuration for robot.conf
+# This file must be at /var/lib/robot/config.bash
```

Comment:
I have designed this as a configuration file for each robot. So it is intened to modify this file according to robot configuration. So I have to put it outside of git repository.

And this file will be used from multiple users since robot configuration will be the same from users. So I put it under `/var/lib/robot/` as like other configuration files are (e.g. `app_schedules.yaml`, `switchbot_token.txt`, `twitter_account_fetch_jsk.yaml` and so on)